### ISSUE TITLE
Testing pre-trained model on custom data

### ISSUE THREAD

**Issue by sownds7** [HRNet/HRFormer#8]
Hi, 

I am trying test pretrained models (ade20k segmentation model) on custom data but the default loader seems to not read the images files in label path even when the data in available in the file path.

Kindly advise how test on custom data. Also please mention the file structure needed for the data testing.

Appreciate your help. 

Thankyou!
